update translation info in readme.md (#179)
tlovett1/safe-redirect-manager: README.md

plugin uploader ("Plugins" > "Add New" > "Upload Plugin").

## Non-English Usage

> Safe Redirect Manager is available in English and other languages.  A listing of those languages and instructions for translating the plugin into other languages is available on [Translating WordPress](https://translate.wordpress.org/projects/wp-plugins/safe-redirect-manager/).  Many thanks to the [contributors on the translation teams](https://translate.wordpress.org/projects/wp-plugins/safe-redirect-manager/contributors/)!

Reviewer: ```suggestion
A listing of languages Safe Redirect Manager is available in beyond English and instructions for translating the plugin into more languages is available on [Translating WordPress](https://translate.wordpress.org/projects/wp-plugins/safe-redirect-manager/).  Many thanks to the [contributors on the translation teams](https://translate.wordpress.org/projects/wp-plugins/safe-redirect-manager/contributors/)!
```

Reviewer: I feel like my edit is still structurally awkward, not sure how to fix it at the moment but the current initial sentence is also strange with the heading IMO.
